src/pykiso/cli.py

```diff
@@ -94,6 +95,18 @@ def check_file_extension(
     return paths
 
 
+def get_file_name_without_extension(path: PathType) -> str:
+    """Get name of a file without its extension
+
+    :param path: path of the file
+
+    :return: file name without extension
+    """
+    full_name = os.path.basename(path)
```

Comment:
With pathlib this would become:
```return Path(path).resolve().name```

Reply:
Actually Path(config_file).stem does exactly what I want and this whole function is useless 🤦‍♂️

Reply:
this function will be then removed ?

Reply:
done